src/contracts/FOXy.sol

```diff
@@ -84,8 +84,6 @@ contract Foxy is ERC20Permit, Ownable {
         @return bool
      */
     function setIndex(uint256 _index) internal returns (bool) {
-        // make sure index isn't set yet, so it's only set once
-        require(index == 0, "Index already set");
```

Comment:
didn't need since setIndex is internal now

Reply:
I think the reason they seperated this was to allow migration to OHMv2, fine for now, but just worth noting.